src/content-helper/common/utils/date.ts

```
@@ -68,6 +73,10 @@ export function getSmartShortDate( date: Date ): string {
  * @return {string} The resulting date in "YYYY-MM-DD" format.
  */
 export function removeDaysFromDate( date: string, days: number ): string {
+	if ( false === Number.isInteger( days ) ) {
+		return __( 'days parameter must be an integer', 'wp-parsely' );
```

Instead of returning `string` should we throw `UnexpectedValueException` here?

---

We could, though I don't necessarily think we should since we're able to show the error 🤔